app/scripts/router.js

```diff
@@ -137,10 +137,8 @@ function (
       this.formPrefill = options.formPrefill;
       this.notifications = options.notifications;
       this.able = options.able;
+      this.storage = Storage.factory('sessionStorage', this.window);
```

Comment:
Probably unhelpful but, imho, using naming conventions to imply privacy rather than enforcing it using closures is the real source of your grief here. :)